test/cmd/quota.sh

```diff
@@ -4,6 +4,16 @@ trap os::test::junit::reconcile_output EXIT
 
 os::test::junit::declare_suite_start "cmd/quota"
 
+# Cleanup cluster resources created by this test suite
```

Comment:
@stevekuznetsov There are not many places where we delete the namespace before starting the suite, one of them is the `templates.sh` test and we do `wait_for_command` there. But I agree if it goes fine without the wait we better remove it. Done. Anything else?